release/release.py

```diff
@@ -26,9 +26,9 @@ def get_version() -> str:
     with pathlib.Path(release_folder / "pyproject.toml").open(mode="rb") as py_project_toml_fp:
         py_project_toml = tomllib.load(py_project_toml_fp)
     version_re = py_project_toml["tool"]["bumpversion"]["parse"]
-    version_tags = [tag for tag in repo.tags if re.match(version_re, tag.tag.tag.strip("v"), re.MULTILINE)]
+    version_tags = [tag for tag in repo.tags if tag.tag and re.match(version_re, tag.tag.tag.strip("v"), re.MULTILINE)]
```

Comment:
Is this to get around the issue of unpushed local changes?

Reply:
No, it's just a typing thing. Apparently `tag.tag` can be None.